dialoguekit/core/dialogue.py

```diff
@@ -9,15 +9,24 @@
 
 
 class Dialogue:
-    def __init__(self, agent_id: str, user_id: str) -> None:
+    def __init__(
+        self, agent_id: str, user_id: str, conversation_id: str
+    ) -> None:
         """Represents a dialogue.
 
         Args:
             agent_id: Agent ID.
             user_id: User ID.
+            conversation_id: Conversation ID.
         """
         self._agent_id = agent_id
         self._user_id = user_id
+        if conversation_id is None:
```

Comment:
`conversation_id` is defined as a positional argument here and in `dialogue_connector.py`, so I am wondering if `conversation_id` should be optional or if this condition can be removed.

Reply:
I made it optional. I think that it will be much easier to maintain it this way.